packages/backend/parser/parser.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+)
+
+const fileLocation = "../../../example/mysa-carbon-log.txt"
+
+func readerCheck(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
+func openAndReadFile(fileName string) string {
+	file, err := os.Open(fileName)
+	readerCheck(err)
+	lines, err := ReadFile(file)
+	readerCheck(err)
+	return lines
+}
+
+// ReadFile takes in a buffer and reads it line by line
+func ReadFile(reader io.Reader) (string, error) {
+	// TODO: Find way to remove declaring error
+	var err error
```

Comment:
`// TODO: Find way to remove declaring error` - Can't you add the types to the return above? I think that means you don't have to declare like this